tests/common/models.py

```diff
@@ -74,7 +74,48 @@ def __init__(self, num_channels: int = 3, num_classes: int = 2) -> None:
         )
         super().__init__(module=net)
 
-        # bind these to class for access during
-        # surgery tests
+        # bind these to class for access during surgery tests
         self.conv1 = conv1
         self.conv2 = conv2
+
+
+class ConvModel(ComposerClassifier):
+    """Convolutional network featuring strided convs, a batchnorm, max pooling, and average pooling."""
+
+    def __init__(self):
+        conv_args = {'kernel_size': (3, 3), 'padding': 1}
+        conv1 = torch.nn.Conv2d(in_channels=32, out_channels=8, stride=2, bias=False, **conv_args)  # stride > 1
+        conv2 = torch.nn.Conv2d(in_channels=8, out_channels=32, stride=2, bias=False,
+                                **conv_args)  # stride > 1 but in_channels < 16
+        conv3 = torch.nn.Conv2d(in_channels=32, out_channels=64, stride=1, bias=False, **conv_args)  # stride = 1
+        bn = torch.nn.BatchNorm2d(num_features=64)
+        pool1 = torch.nn.MaxPool2d(kernel_size=(2, 2), stride=2, padding=1)
+        pool2 = torch.nn.AdaptiveAvgPool2d(1)
+        flatten = torch.nn.Flatten()
+        linear1 = torch.nn.Linear(64, 48)
+        linear2 = torch.nn.Linear(48, 10)
+
+        net = torch.nn.Sequential(
+            conv1,
+            conv2,
+            conv3,
+            bn,
+            pool1,
+            pool2,
+            flatten,
+            linear1,
+            linear2,
+        )
+
+        super().__init__(module=net)
+
+        # bind these to class for access during surgery tests
+        self.conv1 = conv1
+        self.conv2 = conv2
+        self.conv3 = conv3
+        self.bn = bn
+        self.pool1 = pool1
+        self.pool2 = pool2
```

Comment:
This looks scary but I just added `self.` before all the child modules where we create them, instead of creating them and then assigning to attributes later. Looks like more typical torch code + saves space.

Would be nice to just construct all the modules anonymously in the Sequential, but it sounds like that makes the testing harder I assume it's not worth it.

```suggestion
        self.conv1 = torch.nn.Conv2d(in_channels=32, out_channels=8, stride=2, bias=False, **conv_args)  # stride > 1
        self.conv2 = torch.nn.Conv2d(in_channels=8, out_channels=32, stride=2, bias=False,
                                **conv_args)  # stride > 1 but in_channels < 16
        self.conv3 = torch.nn.Conv2d(in_channels=32, out_channels=64, stride=1, bias=False, **conv_args)  # stride = 1
        self.bn = torch.nn.BatchNorm2d(num_features=64)
        self.pool1 = torch.nn.MaxPool2d(kernel_size=(2, 2), stride=2, padding=1)
        self.pool2 = torch.nn.AdaptiveAvgPool2d(1)
        self.flatten = torch.nn.Flatten()
        self.linear1 = torch.nn.Linear(64, 48)
        self.linear2 = torch.nn.Linear(48, 10)

        net = torch.nn.Sequential(
            self.conv1,
            self.conv2,
            self.conv3,
            self.bn,
            self.pool1,
            self.pool2,
            self.flatten,
            self.linear1,
            self.linear2,
        )

        super().__init__(module=net)
```